runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatConfig.java

```diff
@@ -148,4 +149,26 @@ default boolean confirmKicks()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		keyName = "showIgnores",
+		name = "Marks ignored players",
+		description = "Allows you to pick out players that are from your ignore list",
```

Comment:
maybe "Recolors players on your ignore list" or something. "Allows you to pick out" could mean anything.